### DISCUSSION TITLE
Prevent binding an `IArgumentModel` property

### DISCUSSION THREAD

**Discussion by lonix1** [bilal-fazlani/commanddotnet#459] [Q&A] [answered]
I'm using the `IArgumentModel`, and I want to place some stuff in there that should not be bound:
````cs
public class MyModel : IArgumentModel
{
  [Option('f', "foo")]
  public string Foo { get; set; }

  public string Bar => "123" + Foo + "456";     // <<<
}
````

The `Bar` property lacks a setter, so I assumed the library would not attempt to bind it. But I get this:

> CommandDotNet.InvalidConfigurationException: Operand property must be attributed with OperandAttribute or OrderByPositionInClassAttribute to guarantee consistent order. Property: MyProject.MyModel.Bar

Is there some attribute (e.g. the way one uses `[JsonIgnore]` when doing json serialisation) or workaround to ignore that property?

**Comment by drewburlingame** [comment, marked as answer]
We do not have this concept. The intent is to keep the arg models single purposed.  They are only for defining arguments.  I recommend using methods instead of get only properties when you need to perform an operation on the user provided values.
